### DISCUSSION TITLE
请教为啥power页面，reboot命令后面跟着一个冒号(：) ?

### DISCUSSION THREAD

**Discussion by alexlabs** [yume-chan/ya-webadb#411] [Q&A] [answered]
你好，我是一个js小白。有个问题请教您，一般常见的reboot 命令应该是如下格式：adb reboot bootloader 。请教为demo中的power页面，reboot命令后面跟着一个冒号(：) ?

**Comment by upintheairsheep** [comment]
Title Translation:
Why is the reboot command followed by a colon (:) on the power page?

   你好，我是一个js小白。有个问题请教您，一般常见的reboot 命令应该是如下格式：adb reboot bootloader 。请教为demo中的power页面，reboot命令后面跟着一个冒号(：) ?

**Comment by upintheairsheep** [comment]
Translation: 
Hello, I am a js novice. I have a question for you, the general reboot command should be in the following format: adb reboot bootloader . For the power page in the demo, the reboot command is followed by a colon (:) ?

**Comment by yume-chan** [comment, marked as answer]
This is the protocol between ADB client and daemon. It's unrelated to the command line arguments.
